cassava-megaparsec.cabal

```diff
@@ -30,7 +30,7 @@ library
                     , cassava          >= 0.4.2 && < 0.6
                     , megaparsec       >= 7.0   && < 10.0
                     , unordered-containers >= 0.2.7 && < 0.3
-                    , vector           >= 0.11  && < 0.13
+                    , vector           >= 0.11  && <= 0.13
```

Comment:
That's not how you do it!
You want `<0.14` so that you include the minor and patch releases `0.13.x.y`.